docs/topics/wasm/wasm-debugging.md

```diff
@@ -21,7 +21,7 @@ Create a project using the Kotlin Multiplatform wizard:
 3. Select the **Web** option. Make sure that no other options are selected.
 4. Click the **Download** button and unpack the resulting archive.
 
-![Kotlin Multiplatform wizard](wasm-compose-wizard.png){width=600}
+![Kotlin Multiplatform wizard](wasm-compose-web-wizard.png){width=600}
```

Comment:
I think you can cut the width in half, now the wizard seems too big compared to the instruction list

The same is true for all other images on this page that have big lengths. The maximum width makes them too big, taking up the whole page scroll, like in the Custom formatters section

Reply:
Adjusted all images in a size that it's not too big but allows clear visibility.